org.opencpn.OpenCPN.yaml

```
@@ -0,0 +1,96 @@
+# Copyright (c) 2018 Alec Leamas
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+
+# This manifest is used to build the official flathub package. 
+
+
+app-id: org.opencpn.OpenCPN
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
```

Update to runtime 19.08
```suggestion
runtime-version: 19.08
```

---

Obviously, we will have to do this change. However, this introduces an ABI break which not only affects opencpn but also a plugin echosystem. So, this is non-trivial.

The strategy for now is to postpone this until wxWidgets 3.2 is available. This is supposed to be "this spring", whatever that means. This way we could make a clean  break which handles both wxWidgets and the flatpak runtime.

---

I'm not sure if I follow. Have you shipped plugins as Flatpak extensions before? In general runtime shouldn't affect the way you build wxgtk3, it can stay at whatever version you need.

---

My reply was in the wrong place, see https://github.com/flathub/flathub/pull/1528#issuecomment-630609623